Examples/SampleApp/iOS/Info.plist

```diff
@@ -36,5 +36,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string></string>
```

Comment:
Does it work with empty string? Maybe put something generic here, "This app would like you use your location"?